### PR TITLE
Adding NVIDIA SE-ResNeXt

### PR DESCRIPTION
**Pull request by nv-kkudrynski**
@vmoens @soumith Adding another from Nvidia. 
In the meanwhile can we fix my CLA. I did all the formalites on my side and sent the email to cla@fb.com followed by confirmation from my officials. Thanks

### CODE REVIEW

**Comment by vmoens** [comment]
> @vmoens @soumith Adding another from Nvidia. In the meanwhile can we fix my CLA. I did all the formalites on my side and sent the email to [cla@fb.com](mailto:cla@fb.com) followed by confirmation from my officials. Thanks

Hi, I'm chasing them to see what's happening, sorry about that! I'll keep you posted.
Let me review this in the meantime :)
Thanks for the work + patience for this (rather painful) CLA nuisance...

File: nvidia_deeplearningexamples_se-resnext.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    plt.imshow(img)
+    plt.show()
+    print(result)
+
+```
+
+### Details
+For detailed information on model input and output, training recipies, inference and performance visit:
+[github](https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/Classification/ConvNets/se-resnext101-32x4d)
+and/or [NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/resources/se_resnext_for_pytorch)
```
Comment (vmoens) [inline nvidia_deeplearningexamples_se-resnext.md:111]: full stop missing

**Review by vmoens — CHANGES_REQUESTED** [review]

File: nvidia_deeplearningexamples_se-resnext.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    'http://images.cocodataset.org/test-stuff2017/000000004954.jpg',
+]
+
+
+batch = torch.cat(
+    [utils.prepare_input_from_uri(uri) for uri in uris]
+).to(device)
+```
+
+Run inference. Use `pick_n_best(predictions=output, n=topN)` helepr function to pick N most probably hypothesis according to the model.
```
[the remark this answers is not in the document]
Comment (nv-kkudrynski) [reply nvidia_deeplearningexamples_se-resnext.md]: fixed

File: nvidia_deeplearningexamples_se-resnext.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+accelerator: cuda
+order: 10
+---
+
+
+### Model Description
+
+The ***SE-ResNeXt101-32x4d*** is a [ResNeXt101-32x4d](https://arxiv.org/pdf/1611.05431.pdf)
+model with added Squeeze-and-Excitation module introduced
+in [Squeeze-and-Excitation Networks](https://arxiv.org/pdf/1709.01507.pdf) paper.
```
[the remark this answers is not in the document]
Comment (nv-kkudrynski) [reply nvidia_deeplearningexamples_se-resnext.md]: fixed

File: nvidia_deeplearningexamples_se-resnext.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+This model is trained with mixed precision using Tensor Cores on Volta, Turing, and the NVIDIA Ampere GPU architectures. Therefore, researchers can get results 3x faster than training without Tensor Cores, while experiencing the benefits of mixed precision training. This model is tested against each NGC monthly container release to ensure consistent accuracy and performance over time.
+
+We use [NHWC data layout](https://pytorch.org/tutorials/intermediate/memory_format_tutorial.html) when training using Mixed Precision.
+
+#### Model architecture
+
+![SEArch](https://pytorch.org/assets/images/SEArch.png)
```
[the remark this answers is not in the document]
Comment (nv-kkudrynski) [reply nvidia_deeplearningexamples_se-resnext.md:32]: I found out this is where the image lands on your side after the merge. From here it is visible both in the markdown in the hub and in the gcollab (where relative path does not work).

File: nvidia_deeplearningexamples_se-resnext.md (hunk 1 above)
Comment (nv-kkudrynski) [reply nvidia_deeplearningexamples_se-resnext.md]: fixed

File: nvidia_deeplearningexamples_se-resnext.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (vmoens) [reply nvidia_deeplearningexamples_se-resnext.md:32]: Ok got it, it does not appear in netlify though... I guess we can consider it as a netlify issue then!

**Review by vmoens — APPROVED** [review]

**Pull request merged**